src/dmd/ctorflow.d

```diff
@@ -105,53 +105,118 @@ struct CtorFlow
 
 
 /****************************************
- * Merge fi flow analysis results into fieldInit.
+ * Merge `b` flow analysis results into `a`.
  * Params:
- *      fieldInit = the path to merge fi into
- *      fi = the other path
+ *      a = the path to merge `b` into
+ *      b = the other path
  * Returns:
- *      false means either fieldInit or fi skips initialization
+ *      false means one of the paths skips construction
  */
-bool mergeFieldInitX(ref CSX fieldInit, const CSX fi)
+bool mergeCallSuper(ref CSX a, const CSX b) pure nothrow
```

Comment:
Why not call it `leftBranch` and `rightBranch`, if that is what it means?

Reply:
Maybe `lhs` and `rhs` if brevity is really needed?

Reply:
Please, no.

* What follows in the implementation is essentially math, and in math one uses one letter variable names for good reason. We write `sin x`, not `sin theAngleInRadians`.

* One letter names are easiest to comprehend when one has complex math expressions.

* The algorithm is commutative, i.e. there is no `left` or `right`. Giving such names implies there is an asymmetry when there is none.

* Also, it's good style to use shorter names the more local they are.

Reply:
The names are fine. The code is wanting. Reviewers should suggest material improvement, not trivial pursuits.